notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -42,6 +42,26 @@ define([
         this.edit_shortcuts = new keyboard.ShortcutManager(undefined, options.events, this.actions, this.env);
         this.edit_shortcuts.add_shortcuts(this.get_default_common_shortcuts());
         this.edit_shortcuts.add_shortcuts(this.get_default_edit_shortcuts());
+
+
+        this.config = options.config;
+        var that = this;
+
+        this.config.loaded.then(function(){ 
+            
+            (((that.config.data.keys||{}).edit||{})
```

Comment:
expanded, but that look a bit ugly for just avoiding typeerror if one of them is undefined: 

```
 +            try {
 +                edit_unbind = that.config.data.keys.edit.unbind;
 +            } catch (e) {
 +                if (e instanceof TypeError) {
 +                    edit_unbind = [];
 +                } else {
 +                    throw e;
 +                }
 +            }
```